indexers/keyvalue/LevelDBIndexer/__init__.py

```diff
@@ -1,37 +1,40 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from typing import Optional, Iterator, Any
+from typing import Optional, Iterable
 
 from jina import Document
 from jina.executors.indexers.keyvalue import BinaryPbIndexer
 
+if False:
+    from plyvel import DB
+
 
 class LevelDBIndexer(BinaryPbIndexer):
     """
     :class:`LevelDBIndexer` use `LevelDB` to save and query protobuf document.
     """
 
-    def get_add_handler(self):
-        """Get the database handler
+    def get_add_handler(self) -> 'DB':
+        """Get the database handler.
         """
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def get_create_handler(self):
-        """Get the database handler
+    def get_create_handler(self) -> 'DB':
+        """Get the database handler.
         """
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def get_query_handler(self):
-        """Get the database handler
+    def get_query_handler(self) -> 'DB':
+        """Get the database handler.
         """
         import plyvel
         return plyvel.DB(self.index_abspath, create_if_missing=True)
 
-    def query(self, key: Any, *args, **kwargs) -> Optional[Any]:
-        """Find the protobuf chunk/doc using id
+    def query(self, key: str, *args, **kwargs) -> Optional[bytes]:
+        """Find the protobuf documents via id.
```

Comment:
find the serialized? since is what u are adding?

Reply:
check